e2e/integration/journey.test.js

```diff
@@ -113,9 +113,12 @@ it('requires employee selection', () => {
   cy.findByRole('button', { name: /submit/i }).click()
   cy.findByText(/form is incomplete/i).should('exist')
 
-  cy.findByRole('combobox', { name: /who is your employer/i }).select(
-    'Capgemini'
-  )
+  cy.findByRole('textbox', { name: /who is your employer/i }).type('Ca')
+  cy.get("option[class='employer-option']").each(($ele, i, $list) => {
+    if ($ele.text() === 'Capgemini') {
+      cy.wrap($ele).click()
+    }
+  })
```

Comment:
You should still be able to do this with Testing Library selectors, e.g.:
```suggestion
  cy.findByRole('option', { name: /capgemini/i }).click()
```

Reply:
applied the required changes